metaci/release/models.py

```diff
@@ -1,13 +1,20 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import re
+
 from django.db import models
+from django.utils.dateparse import parse_date
+from django.utils.dateparse import parse_datetime
 from django.utils.translation import ugettext_lazy as _
 
 from model_utils import Choices
 from model_utils.models import StatusModel
 from model_utils.fields import AutoCreatedField, AutoLastModifiedField
 
+from metaci.release.utils import update_release_from_github
+from metaci.repository.utils import get_github_api
```

Comment:
@jlantz There are still some unused imports. Install pylint in your virtualenv and VSCode should show errors about them.